src/main/kotlin/no/nav/familie/ef/sak/behandling/BehandlingRepository.kt

```diff
@@ -67,7 +67,8 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
         SELECT b.id behandling_id, be.id ekstern_behandling_id, fe.id ekstern_fagsak_id
         FROM behandling b
             JOIN behandling_ekstern be ON b.id = be.behandling_id
-            JOIN fagsak_ekstern fe ON b.fagsak_id = fe.fagsak_id
+            JOIN fagsak_ekstern fe ON b.fagsak_id = fe.fagsak_id 
+        WHERE behandling_id IN (:behandlingId)
         """)
     fun finnEksterneIder(behandlingId: Set<UUID>): Set<EksternId>
```

Comment:
Fint hvis du legger inn empty-sjekk for servicen som bruker denne metoden også, eks 
```kotlin
fun hentEksterneIder(behandlingIder: Set<UUID>) =
            behandlingIder.takeIf { it.isNotEmpty() }?.let { behandlingRepository.finnEksterneIder(it) }
```